lib/debug/session.rb

```diff
@@ -1057,11 +1057,16 @@ def step_command type, arg
 
     def config_show key
       key = key.to_sym
-      if CONFIG_SET[key]
+      if config_detail = CONFIG_SET[key]
```

Comment:
I'm not a fan of doing assignments in `if` statements, could you move assignment to a separate line?

Reply:
Updated